README.md

```diff
@@ -1,4 +1,7 @@
-# yandex_mapkit
+# yandex_mapkit_lite
+
+**This is a simplified version of the yandex_mapkit 3.4.0 package.**
+Features such as Suggestions, Search, ReverseSearch, Bicycle, Driving have been removed from it.
```

Comment:
It is necessary to explicitly mark that this library is a fork.
It is necessary to describe in more detail the difference from the full version of maps (what functions are missing, if they are needed, we recommend using mapkit full), in what cases it should be used. And provide a link to a specific mapkit package.

Reply:
✅